sunscreen_math/src/lib.rs

```diff
@@ -96,3 +101,34 @@ pub(crate) fn ristretto_bitwise_eq(a: RistrettoPoint, b: RistrettoPoint) -> bool
 
     a == b
 }
+
+#[macro_export]
```

Comment:
Oh, I didn't actually want to make this public :/

Reply:
Oh I thought we had talked about others using this? If that's the case, I think the correct thing to do is remove `#[macro_export]` and then below the macro, include the line `pub(crate) use refify;` You'll have to update the `refify` imports because the path will no longer be from the crate root.

Reply:
Ok, just documented this thing.